APMrover2/Parameters.cpp

```diff
@@ -413,6 +413,12 @@ const AP_Param::Info Rover::var_info[] = {
     // @Path: Parameters.cpp
     GOBJECT(g2, "",  ParametersG2),
 
+    #if OSD_ENABLED
```

Comment:
this should be "#if OSD_ENABLED == ENABLED"

Reply:
I wiöö change it ...